pkg/open/open.go

```diff
@@ -29,3 +29,22 @@ func Browser(url string) error {
 
 	return nil
 }
+
+// LacksBrowser determines if no browser is set in linux
+func LacksBrowser() bool {
```

Comment:
I think `Lacks` is a bit difficult to parse the context of this API -- Instead I would consider a `CanOpenBrowser` or `HasDefaultBrowser` a bit more straight forward [but then the return value would be different].